packages/docusaurus-plugin-content-docs/src/index.ts

```diff
@@ -476,23 +473,6 @@ Available document ids=
       );
     },
 
-    async routesLoaded(routes) {
```

Comment:
@teikjun I'm going to put this change in another PR to be able to release a few fixes asap, so please revert it here

Reply:
@slorber Ok, got it!